packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue

```diff
@@ -0,0 +1,230 @@
+<script lang="ts" setup>
+import { OnyxIcon } from "@/index";
+import checkSmall from "@sit-onyx/icons/check-small.svg?raw";
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import type { OnyxSwitchProps } from "./types";
+import { areObjectsFlatEqual } from "@/utils/comparator";
+import { transformValidityStateToObject } from "@/utils/forms";
+import { computed, ref, toRefs, watch } from "vue";
+const props = withDefaults(defineProps<OnyxSwitchProps>(), {
```

Comment:
```suggestion

const props = withDefaults(defineProps<OnyxSwitchProps>(), {
```

I'd like to add a new line :)